### PR TITLE
Float panels

### PR DESCRIPTION
**Pull request by huwshimi**
Make the panels float in preparation for the redesign.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- check the panels in various scenarions
- click on a charm
- open the inspector
- open the machine view
- scroll the list of charms
- do a search

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2732/
Test FAILed.

**Comment by hatched** [comment]
:+1: QA OK

This, looks, awesome :)

File: app/assets/css/_environment-header.scss [hunk 1]
```diff
@@ -1,57 +1,32 @@
 .environment-header {
+    @extend .floating-panel;
```
Comment (anthonydillon) [inline app/assets/css/_environment-header.scss]: A placeholder here would be better. @extending classes can lead to larger then required style sheets.

http://thesassway.com/intermediate/understanding-placeholder-selectors

Comment (huwshimi) [inline app/assets/css/_environment-header.scss]: Ah yes, much better, fixed!

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/2733/
Test FAILed.

**Comment by anthonydillon** [comment]
Great stuff Huw, much nicer :+1: QA ok

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**